solidity/contracts/TransformerOracle.sol

```diff
@@ -45,7 +45,7 @@ contract TransformerOracle is BaseOracle, AccessControl, ITransformerOracle {
 
   /// @inheritdoc ITransformerOracle
   function getMappingForPair(address _tokenA, address _tokenB) public view virtual returns (address _mappedTokenA, address _mappedTokenB) {
-    (ITransformer _transformerTokenA, ITransformer _transformerTokenB) = _getTransformers(_tokenA, _tokenB);
+    (ITransformer _transformerTokenA, ITransformer _transformerTokenB) = _getTransformers(_tokenA, _tokenB, true, true);
```

Comment:
Is it worth having `getMappingForPair` and `getRecursiveMappingForPair`?

Reply:
It might help for debugging purposes, and honestly, I think it doesn't heart